index.js

```diff
@@ -70,11 +70,13 @@ module.exports = function(source) {
   });
 
   var boundResolvers = PathCache.resolvers(options, this.resolve);
-  PathCache.createFromFile(boundResolvers, {}, options.filename)
+  PathCache.createFromFile(boundResolvers, {}, source, options.filename)
     .then(function(importPathCache) {
       // CachedPathEvaluator will use this PathCache to find its dependencies.
       options.cache = importPathCache;
-      importPathCache.allDeps().forEach(self.addDependency);
+      importPathCache.allDeps().forEach(function (f) {
+        self.addDependency(path.normalize(f));
```

Comment:
Kind of surprised we haven't gotten this before from other windows fixes. Thanks.